src/main/webapp/student/scripts/play-game.js

```diff
@@ -58,10 +61,20 @@ function getGameInstanceIdFromQueryParams() {
   return urlParams.get('gameInstanceId');
 }
 
+function resetDOM() {
+  selectedAnswerId = "";
+  submited = false;
+  document.getElementById("result").innerText = '';
+  if(isCorrect != null) {
+  resultObject.classList.toggle(isCorrect.toString());
+  }
+  isCorrect = null;
+  document.getElementById("gif").setAttribute('src','')
+}
+
 // Inits listener to User's points in Firestore DB
-function initPointsListener(gameInstanceId, user) {
-  db.collection("gameInstance").doc(gameInstanceId).collection("students").doc(user.uid)
-  .onSnapshot(function(doc) {
+function updatePoints() {
+  db.collection("gameInstance").doc(gameInstanceId).collection("students").doc(studentId.uid).get().then(function(doc) {
```

Comment:
Well thought for the points! Nice that now its queried when the question is finished and not before.